src/frequenz/sdk/_internal/_channels.py

```diff
@@ -37,14 +41,35 @@ def __init__(self, receiver: Receiver[T]) -> None:
             receiver: The receiver to cache.
         """
         self._receiver = receiver
-        self._latest_value: T | None = None
+        self._latest_value: T | _Sentinel = _Sentinel()
         self._task = asyncio.create_task(self._run())
 
     @property
-    def latest_value(self) -> T | None:
-        """Get the latest value in the cache."""
+    def latest_value(self) -> T:
+        """The latest value that has been received.
+
+        This raises a `ValueError` if no value has been received yet. Use `has_value` to
+        check whether a value has been received yet, before trying to access the value,
+        to avoid the exception.
+
+        Returns:
+            The latest value that has been received.
+
+        Raises:
+            ValueError: If no value has been received yet.
+        """
+        if isinstance(self._latest_value, _Sentinel):
+            raise ValueError("No value has been received yet.")
```

Comment:
I personally found it weird that a property raises. Maybe it should no longer be a property?

Reply:
That's true,  I've changed it to the `get` method.  That seems to go well with the other new `has_value` method as well.

Reply:
cool, thanks! It does not need an entry into the release-notes because it is internal, right?

Reply:
yup, it is not exposed outside at the moment.